doc/batcher-api.md

```diff
@@ -0,0 +1,143 @@
+# Batcher API Guide
+
+The batcher has two responsabilities: choosing which aggregator to choose for a metric instrument and store the last record for each metric ready to be exported.
+
+## Selecting a specific aggregator for metrics
+
+Sometimes you may want to use a specific aggregator for one of your metric, export a average of the last X values instead of just the last one.
```

Comment:
```suggestion
Sometimes you may want to use a specific aggregator for one of your metric, export an average of the last X values instead of just the last one.
```